app/views/organizations/staff/pets/index.html.erb

```diff
@@ -42,7 +42,7 @@
       <div class="card">
         <!-- table -->
         <div class="table-responsive overflow-y-hidden">
-          <table class="table mb-0 text-nowrap table-hover table-centered">
+          <table style="height: 180px;" class="table mb-0 text-nowrap table-hover table-centered">
             <thead>
               <tr>
                 <th scope="col">Name</th>
```

Comment:
```suggestion
          <table class="table mb-0 text-nowrap table-hover table-centered">
            <thead>
              <tr style="height: 70px;">
                <th scope="col">Name</th>
```

I'm thinking this will give just enough room for the dropdown with having the least amount of changes to the design. Thoughts?

Reply:
Good idea, 
![image](https://github.com/rubyforgood/pet-rescue/assets/51760107/bec72316-3daf-4c12-a0e4-dca0d9ecc6b7)
![image](https://github.com/rubyforgood/pet-rescue/assets/51760107/85d5ec4e-0a3a-488c-abeb-52f855ebecbb)